src/app/pages/component-viewer/_component-viewer-theme.scss

```diff
@@ -0,0 +1,57 @@
+@mixin component-viewer-theme($theme) {
+  $primary: map-get($theme, primary);
+  $accent: map-get($theme, accent);
+  $warn: map-get($theme, warn);
+  $background: map-get($theme, background);
+  $foreground: map-get($theme, foreground);
+
+  app-component-viewer {
+    color: md-color($foreground, secondary-text);
+
+    .docs-component-viewer-tabbed-content {
+      h1, h4 {
+        color: md-color($primary, 800);
+        background: rgba(md-color($foreground, secondary-text), .03);
+      }
+
+      h3, h2, h5, p, ol, li{
```

Comment:
can you target classes rather than tag names, here and elsewhere. my concern with targeting tag names is that the styles may affect the code examples (if these are elements generated by djnei, jeremy can add classes for you)